src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -262,7 +264,7 @@ void uartIrqHandler(uartPort_t *s)
 
 static void handleUsartTxDma(uartPort_t *s)
 {
-    // DMA_Cmd(s->txDMAStream, DISABLE);
+    DMA_Cmd(s->txDMAStream, DISABLE);
```

Comment:
Hmm... FEIF is only checked if TCIF... means handleUsartTxDma is always called with disabled???

Gee, I'm getting really confused now. Must read the entire DMA section again...